lib/derivativo/conversion.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'benchmark'
+
+module Derivativo
+  module Conversion
+    def self.image_to_image(src_file_path:, dst_file_path:, rotation: '0')
+      with_logged_timing(__callee__, src_file_path, dst_file_path) do
+        Imogen.with_image(src_file_path) do |img|
+          Imogen::Iiif.convert(
+            img,
+            dst_file_path,
+            File.extname(dst_file_path)[1..-1],
```

Comment:
More of an FYI than a suggestion:

Assuming the [1..-1] is used to remove the '.' from the extension, delete_prefix('.') can also be used. One advantage: it does not complain if used on an empty string.

Reply:
Good suggestion, thanks!